bin/git-theta

```diff
@@ -108,6 +124,19 @@ def track(args):
     git_utils.add_file(gitattributes_file, repo)
 
 
+def add(args, unparsed_args):
```

Comment:
What is the reason for manually using subprocess.run to talk to git over the GitPython `repo.git.add` command? 

It looks like already has an environment variable context manager, from the docs:

```python
ssh_cmd = 'ssh -i id_deployment_key'
with repo.git.custom_environment(GIT_SSH_COMMAND=ssh_cmd):
    repo.remotes.origin.fetch()
```

Reply:
Good call, simpler to just use gitpython.